### PR TITLE
ci: test on windows targets

### PR DESCRIPTION
**Pull request by mattsse**
## Motivation
Build on windows targets in CI.
But only builds, for tests, need to figure out how to install ganache and geth on windows

## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Updated the changelog

### CODE REVIEW

**Review by gakonst — APPROVED** [review]
good call - would've caught the Windows incompatibility on svm-rs earlier

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -222,3 +222,53 @@ jobs:
         run: |
             export PATH=$HOME/bin:$PATH
             ./scripts/examples.sh
+
+  windows-build:
```
Comment (gakonst) [inline .github/workflows/ci.yml:226]: Might be worth putting this in a different workflow file?

Comment (mattsse) [reply .github/workflows/ci.yml:226]: windows.yml?

Comment (mattsse) [reply .github/workflows/ci.yml:226]: but I'm a little hesitant to move this out, perhaps we can leverage all the geth, ganach stuff and unify everything in a matrix build with all vars etc, like I've attempted here https://github.com/gakonst/foundry/pull/218

**Comment by gakonst** [comment]
Blocked on https://github.com/roynalnaruto/svm-rs/pull/9 getting merged / new release

**Review by gakonst — APPROVED** [review]

**Pull request merged**